### PR TITLE
Configured yarn (or npm). Updated README.md accordingly.

### PR DESCRIPTION
**Pull request by tralves**
Created the yarn (or npm) `package.json` file with the proper dependencies.

Contributing to this project just got (a bit) easier!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,22 +13,36 @@ This is a fork from that code, as it was at 11/11/2013.
...
 https://github.com/marado/TalkerNode .
 
--------------------------------------------------------------------------
+## Installation
 
-sam says: teh SamLicence is basically "Do as you will - but if you make
-millions can I get a pint please?"
+```bash
+> git clone https://github.com/marado/TalkerNode
+> yarn # or `npm install`, if you're still in, like, two weeks ago... pffft...
```
Comment (marado) [inline README.md]: # or `npm install`
we don't need the npm users dissing, specially when, until this PR, I *was* two weeks ago... ;-)

**Comment by marado** [comment]
@tralves I made you a PR with the changes from the comments. If you could merge it and regenerate the yarn.lock file, that would be great.

**Comment by marado** [comment]
BTW, in case you didn't know: one more PR and you deserve a tshirt!
https://hacktoberfest.digitalocean.com/

File: README.md (hunk 1 above)
Comment (tralves) [inline README.md]: I was a npm user two _hours_ ago :)
It was a joke. Just making sure you read the README.md... :)

**Comment by tralves** [comment]
@marado of course I know about hacktoberfest. Why do you think I started contributing _now_?

J/K... I wanted to get into this for a long time. Hacktoberfest just gave the final push :)

**Comment by marado** [comment]
I'm going ahead already and merge this, since that .dig (last commit) needs Nodiverse 0.2.1 and I want to bump that dependency version ASAP. I'll then pull my patches on top of it. However, this doesn't mean our conversation needs to be over :-)

**Pull request merged**